### PR TITLE
Adding pardiso

### PR DESCRIPTION
**Pull request by ma-sadeghi**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] ~~If static libraries are linked in, the license of the static library is packaged.~~
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by ma-sadeghi** [comment]
Hi @conda-forge/help-python. What does the 3rd item on the checklist mean? 

>  - [ ] Source is from official source

Just to clarify, this package is a fork of the original repo, i.e. `https://github.com/haasad/PyPardisoProject`. The reason I didn't use the main repo is that I've contacted the author several times and it seems that he/she's no longer active on GitHub and the most recent change dates back to 2016. I want to make some changes to the package soon, and there's no way of contributing to the original repo, so that's why I forked the project. Is that OK?

File: recipes/pypardiso/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+{% set name = "pypardiso" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # uncomment the line below, and modify as needed. Use releases if available:
```
Comment (synapticarbors) [inline recipes/pypardiso/meta.yaml:9]: remove comments throughout recipe

File: recipes/pypardiso/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+build:
+  noarch: python
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
```
Comment (synapticarbors) [inline recipes/pypardiso/meta.yaml:26]: you don't need this `build` section.

**Comment by synapticarbors** [comment]
I don't have a strong opinion about using a fork of the repo, but maybe other members of @conda-forge/staged-recipes have comments.

**Comment by ma-sadeghi** [comment]
@synapticarbors Thanks for your review. I removed the build section, but the Windows build seems to be breaking since. Any idea why that's happening?

**Comment by synapticarbors** [comment]
@ma-sadeghi if you look at the logs, it's actually failing due to a different recipe (`e3sm_diags`). You could merge in `master` from upstream since I think the offending recipes have been removed since.

**Comment by ma-sadeghi** [comment]
@synapticarbors Thanks! It seems to be passing.

@conda-forge/staged-recipes Could you please comment on using a fork of the repo? As I briefly explained, the author is no longer active on GitHub, so there's no way of further contributing to the package. In fact, I'm working on a patch that makes `pypardiso` 2X faster for symmetric matrices, which is why I had to fork the repo.

**Comment by isuruf** [comment]
If you are forking, you should rename the fork.

**Comment by ma-sadeghi** [comment]
@isuruf You mean the name of the repo or the library name? In this case, the original repo is named `PyPardisoProject` and the library is called `pypardiso`. Which one should I rename?

**Comment by isuruf** [comment]
Ideally both, but especially the repo and conda package name. Library name too if that's possible.

**Comment by ma-sadeghi** [comment]
@isuruf I changed the fork name from `PyPardisoProject` to `pardiso`, so as the package name from `pypardiso` to `pardiso`, but left the library name as `pypardiso` for now (if that's OK).

Apart from that, is there anything else that needs to be addressed for this PR to be complete?

**Comment by ma-sadeghi** [comment]
Hi @conda-forge/help-python. Could you please review this PR? Thanks!

**Comment by ma-sadeghi** [comment]
@isuruf The C library is called `mkl_pardiso`, and is part of the `mkl`, so there's no standalone `pardiso` package.

**Comment by ma-sadeghi** [comment]
It's a proprietary library and is not publicly distributed (not sure if that makes a difference?). The exact name in the `.so` file is `pardiso600` (https://www.pardiso-project.org).

Quick question: By conflict, do you mean in the context of conda-forge namespace?

**Comment by ma-sadeghi** [comment]
@isuruf Another question: since `pypardiso` is BSD-licenced, what's wrong with packaging the fork as `pypardiso`? (I'm asking since `pypardiso` is the ideal name with the least amount of confusion)

**Comment by isuruf** [comment]
> Quick question: By conflict, do you mean in the context of conda-forge namespace?

Yes

> Another question: since pypardiso is BSD-licenced, what's wrong with packaging the fork as pypardiso?

Whatever you want to call is fine, but the name of the conda-forge package has to be something that people expect. Packaging a fork as `pypardiso` is wrong.

**Comment by ma-sadeghi** [comment]
> > Quick question: By conflict, do you mean in the context of conda-forge namespace?
> 
> Yes
> 
> > Another question: since pypardiso is BSD-licenced, what's wrong with packaging the fork as pypardiso?
> 
> Whatever you want to call is fine, but the name of the conda-forge package has to be something that people expect. Packaging a fork as `pypardiso` is wrong.

@isuruf Fair enough, I renamed the repo to `pardiso4py` to avoid confusion. Please let me know if there's anything else I should modify.

**Comment by ma-sadeghi** [comment]
@isuruf Thank you so much for taking the time. I really appreciate it.

**Pull request merged**